README.md

```diff
@@ -193,8 +193,16 @@ The following sections provide instructions for these combinations:
 
 #### CUDA Build Instructions (NVIDIA only)
 
-**Build instructions are not available yet.  They will be provided at a later
-date**.
+Use the same instructions for fetching and installing both the NVIDIA driver,
+build tools and a copy of `gominer`'s source code.
```

Comment:
Instead of adding a reference here, maybe I should generalize the generic bits of the instructions for both OpenCL and CUDA bits at an earlier section?

Reply:
Yeah, it probably makes sense to split it as amd/nvidia, then have a common nvidia prereqs section and break out the nvidia opencl / cuda specific bits.